.github/workflows/chromatic.yml

```diff
@@ -1,14 +1,17 @@
 name: 'Chromatic'
-on: push
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Does this only run the tests on closed PRs?

Reply:
Ah no my bad this is the Chromatic job. Out of curiosity, why did that have to be changed?

Reply:
I changed it so that it only published when a Pull Request has been merged, see the additional if check.
This is a optimisation separate from the e2e testing. We never really use the storybook generated per push, it seems like overkill.